addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -571,6 +571,7 @@ export class WebsiteSnippetsMenu extends weSnippetEditor.SnippetsMenu {
      * @param {OdooEvent} ev
      */
     _onGMapAPIRequest(ev) {
+        console.log("Event called on the menu", ev);
```

Comment:
Was this commit held for a reason? Should it be squashed?

Reply:
Solved and squashed